src/utils/getAttributeValue.ts

```diff
@@ -15,6 +15,16 @@ function getAttributeValue(node: AST.VAttribute | AST.VDirective) {
       return node.value.expression.value;
     }
 
+    if (node.value.expression.type === 'LogicalExpression') {
+      const operator = node.value.expression.operator;
+      // @ts-ignore
+      const leftSideOfOperation = node.value.expression.left.value;
+      // @ts-ignore
+      const rightSideOfOperation = node.value.expression.right.value;
```

Comment:
The type is not having the value, this might need an update on https://github.com/vuejs/vue-eslint-parser/blob/4587eef81a037ae5e48ec96de34f3adffa1d4151/src/ast/nodes.ts#L364

Reply:
Ok, so I checked this and actually value is available on ESLintLiteral type, which is set as one of the types of ESLintExpression that is the type of the left and right. 

N

ot sure why it is not being recognized here 🤔 
https://github.com/vuejs/vue-eslint-parser/blob/4587eef81a037ae5e48ec96de34f3adffa1d4151/src/ast/nodes.ts#L395

Reply:
Was missing type check of the left and right expressions.